schemas/README.md

```diff
@@ -4,21 +4,29 @@ Generated by https://github.com/foxglove/schemas
 
 ## Contents
 
+- [enum LineType](#enum-linetype)
 - [enum LogLevel](#enum-loglevel)
 - [enum NumericType](#enum-numerictype)
 - [enum PointsAnnotationType](#enum-pointsannotationtype)
 - [enum PositionCovarianceType](#enum-positioncovariancetype)
+- [enum SceneEntityDeletionType](#enum-sceneentitydeletiontype)
+- [ArrowPrimitive](#arrowprimitive)
 - [CameraCalibration](#cameracalibration)
 - [CircleAnnotation](#circleannotation)
 - [Color](#color)
 - [CompressedImage](#compressedimage)
+- [CubePrimitive](#cubeprimitive)
+- [CylinderPrimitive](#cylinderprimitive)
```

Comment:
I'll let others weigh in on this one. For one minor point, the shorter names are more easily confused in code for other types/variables that don't come from @foxglove/schemas.

Reply:
I don't feel strongly since most code will likely access it via a namespaced `foxglove` type or package. Notably we use `CircleAnnotation` to denote that specific circle is unique to annotation. As long as we don't immediately see a value in some other `Cube` message then dropping Primitive seems ok. Keeping Primitive seems equally fine. In a more developed world I'd possibly argue for `foxglove.scene.Cube` 🤣  but beyond that Cube or CubePrimitive is fine.